pkg/kube/exec.go

```diff
@@ -59,6 +60,43 @@ func Exec(cli kubernetes.Interface, namespace, pod, container string, command []
 // returning stdout, stderr and error. `options` allowed for
 // additional parameters to be passed.
 func ExecWithOptions(kubeCli kubernetes.Interface, options ExecOptions) (string, string, error) {
+	config, err := LoadConfig()
+	if err != nil {
+		return "", "", err
+	}
+	o, e, errCh := execStream(kubeCli, config, options)
+
+	var stdout []byte
+	var oerr error
+	wg := sync.WaitGroup{}
+	wg.Add(3)
+	go func() {
+		stdout, oerr = ioutil.ReadAll(o)
+		wg.Done()
+	}()
+
+	var stderr []byte
+	var eerr error
+	go func() {
+		stderr, eerr = ioutil.ReadAll(e)
+		wg.Done()
+	}()
+	go func() {
+		err = <-errCh
+		wg.Done()
+	}()
+	wg.Wait()
+	// Add more info to err
+	if oerr != nil {
+		err = errors.Wrapf(err, "Failed to read stdout: %s", oerr.Error())
+	}
+	if eerr != nil {
+		err = errors.Wrapf(err, "Failed to read stderr: %s", eerr.Error())
+	}
```

Comment:
This could make error with duplicate error statements. We can remove it or just log it in debug mode.